Userland/Services/SpiceAgent/SpiceAgent.h

```diff
@@ -39,22 +32,21 @@ class SpiceAgent {
         TRY(message.write_to_stream(message_stream));
 
         // Create a header to be sent
-        auto header_stream = AK::AllocatingMemoryStream();
-        auto header = TRY(MessageHeader::create(message.type(), message_stream.used_buffer_size()));
-        TRY(header.write_to_stream(header_stream));
+        auto message_header_stream = AK::AllocatingMemoryStream();
+        auto message_header = TRY(MessageHeader::create(message.type(), message_stream.used_buffer_size()));
+        TRY(message_header.write_to_stream(message_header_stream));
 
-        // TODO(Caoimhe): `ChunkHeader` class?
+        // The length given in the chunk header is the length of the message header, and the message combined
+        auto length = message_header_stream.used_buffer_size() + message_stream.used_buffer_size();
 
         // Currently, there are no messages from the agent which are meant for the server.
         // So, all messages sent by the agent with a port of Port::Server get dropped silently.
```

Comment:
This comment looks pretty silly now that `enum class Port` doesn't exist anymore. Perhaps it shouldn't have been added?

Reply:
It does exist in the form of ``ChunkHeader::Port`, and it wasn't added by me, that was always there.